W-12467261 | Release Edifact connector 2.8.0

sms-ebarbeito opened the pull request:
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

sms-ebarbeito committed:
copy resources from folder '2.7' to '2.8'

sms-ebarbeito committed:
add new properties and change an screenshot of config -> identity params to match new version

dhirsch1 committed:
Minor fixes

dhirsch1 committed:
Very minor fixes

sms-dchen commented on edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc:
```diff
@@ -0,0 +1,522 @@
+* false
++
+The connector either uses a substitution character if one is set, or passes the disallowed characters through. |  true |
+| Use Functional Groups | Boolean a| Version of the group message inside functional group envelopes (UNG - UNE):
```
```suggestion
| Use Functional Groups | Boolean a| Group messages with the same functional group identifier inside UNG - UNE envelopes:
```
I believe this description fits better.

dhirsch1 committed:
Update edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc

Co-authored-by: David Chen <76698785+sms-dchen@users.noreply.github.com>

dhirsch1 committed:
Changed field names

jepereira-dev committed:
W-12467261 | Update config parameters
changes:
 - fixed UNG02 and UNG03 field names in the Reference document to match standard names.
 - updated Identity params image.
 - updated Control Numbers image.
 - added functional group parameters in the Config Topics document.
 - added missed ControNumberKey parameters in the Config Topics document.
 - added group map information in the Index document.

dhirsch1 committed:
Added new schema properties, made some small wording tweaks

dhirsch1 committed:
Entered additional changes

dhirsch1 committed:
Small fixes

valkyrie69 commented on edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc:
```diff
@@ -0,0 +1,527 @@
+
+*  On the receive side, the UNB interchange recipient identification code qualifier must match this value. 
+* On the send side, the connector uses this value as the UNB interchange sender identification code qualifier unless it is overridden in either the send configuration or the message parameters. |  |1
+| Mule application sender/recipient ID | String a|  Identification for Mule in the exchange (UNG segment). This value is used in combination with the value of the *Mule application sender/recipient ID qualifier* field, and if one field is set, the other must also be set.
```
```suggestion
| Mule application sender/recipient ID | String a|  Identification for the Mule app in the exchange (UNG segment). This value is used in combination with the value of the *Mule application sender/recipient ID qualifier* field, and if one field is set, the other must also be set.
```

valkyrie69 replied:
? Not sure what "the Mule" is referring to.

valkyrie69 commented on edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc:
```diff
@@ -0,0 +1,527 @@
+
+* On the receive side, the UNG group recipient identification must match this value.
+* On the send side, the connector uses this value as the UNG group sender identification unless it is overridden in the message parameters. |  |
+| Mule application sender/recipient ID qualifier | String a|  Identification qualifier for Mule in the exchange (UNG segment). This value is used in combination with the value of the *Mule application sender/recipient ID* field, and if one field is set, the other must also be set.
```
Same comment as above. "the Mule"?
I wonder why the field name isn't showing up as bold.

valkyrie69 commented on edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc:
```diff
@@ -0,0 +1,527 @@
+
+* On the receive side, the UNG group sender identification must match this value.
+* On the send side, the connector uses this value as the UNG group recipient identification code unless it is overridden in either the send configuration or the message parameters. |  |
+| Parter application sender/recipient ID qualifier | String a|  Identification qualifier for the partner in the exchange (UNG segment). This value is used in combination with the value of the *Partner application sender/recipient ID* field, and if one field is set, the other must also be set.
```
```suggestion
| Partner application sender/recipient ID qualifier | String a|  Identification qualifier for the partner in the exchange (UNG segment). This value is used in combination with the value of the *Partner application sender/recipient ID* field, and if one field is set, the other must also be set.
```

valkyrie69 commented on edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc:
```diff
@@ -0,0 +1,160 @@
+|outputEdiMimeType="APPLICATION_PLAIN"/"APPLICATION_EDIFACT" |Output MIME type of the message.
+|writeInterchangeControlNumberKey="" |Interchange number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for interchange control numbers.
+|writeGroupControlNumberKey="" |Group control number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for group control numbers.
+|writeMessageControlNumberKey="" |Message control number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for message control number.
```
```suggestion
|writeMessageControlNumberKey="" |Message control number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for message control numbers.
```

dhirsch1 committed:
Update edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc

Co-authored-by: Valkyrie Hunter <36547707+valkyrie69@users.noreply.github.com>

dhirsch1 committed:
Update edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc

Co-authored-by: Valkyrie Hunter <36547707+valkyrie69@users.noreply.github.com>

dhirsch1 committed:
Update edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc

Co-authored-by: Valkyrie Hunter <36547707+valkyrie69@users.noreply.github.com>

dhirsch1 committed:
Entered doc edits

dhirsch1 committed:
Merge branch 'latest' into DOCS-W-12467261-EDIFACT-2.8
